src/js/variant-widget/eva-variant-widget.js

```diff
@@ -645,33 +645,14 @@ EvaVariantWidget.prototype = {
             variantStatsPanel.clear(true);
         });
 
+
+
         this.on("variant:change", function (e) {
             if (_.isUndefined(e.variant)) {
                 variantStatsPanel.clear(true);
             } else {
                 if (target.id === _this.selectedToolDiv.id) {
-                    var variant = e.variant;
-                    var region = variant.chromosome + ':' + variant.start + '-' + variant.end;
-                    var proxy = _.clone(this.variantBrowserGrid.store.proxy);
-                    EvaManager.get({
-                        category: 'segments',
-                        resource: 'variants',
-                        query: region,
-                        params: {species: proxy.extraParams.species, studies: proxy.extraParams.studies},
-                        async: false,
-                        success: function (response) {
-                            try {
-//                                _.extend(variant, response.response[0].result[0]);
-                                var result = _.findWhere(response.response[0].result, {start: variant.start,end:variant.end});
-                                _.extend(variant,result);
-                            } catch (e) {
-                                console.log(e);
-                            }
-                        }
-                    });
-                    if (variant.sourceEntries) {
-                        variantStatsPanel.load(variant.sourceEntries, {species: proxy.extraParams.species},_this.studies);
-                    }
+                    _this.loadBottomPanel(variantStatsPanel, e.variant);
```

Comment:
Could this panel variable be renamed to `variantFilesPanel` to avoid confusing it with `variantPopulationStatsPanel`? It doesn't really contain statistics any more either.